app/src/layouts/map/maps.vue

```diff
@@ -233,31 +233,7 @@ import { useRelationsStore } from '../../stores/';
 
 import i18n from '../../lang';
 import { wrap, proxy, Remote } from 'comlink';
-
-function isObject(obj: any): boolean {
-	return typeof obj == 'object' && obj != null;
-}
-function isArray(obj: any): boolean {
-	return Array.isArray(obj);
-}
-
-function objectMap<T extends Object>(object: T, fun: (v: T[keyof T]) => T[keyof T]) {
-	return Object.keys(object).reduce((result: T, key: unknown) => {
-		result[key as keyof T] = fun(object[key as keyof T]);
-		return result;
-	}, {} as T);
-}
-function clone<A>(a: A): A {
-	// @ts-ignore
-	return !isObject(a) ? a : isArray(a) ? a.map(clone) : objectMap(a, clone);
-}
-
-function assign(a: any, b: any): any {
-	if (![a, b].every(isObject)) return (a = b);
-	if ([a, b].every(isArray)) return a.concat(b);
```

Comment:
This line is why I didn't use `lodash.merge`. When mergeing two arrays, they need to be concatenated but instead lodash merge their values by index.

Reply:
When do we have the case that `userLayers.value` is an array instead of an object? I would like to keep everything as simple as possible.

Reply:
Oh, nevermind. It was only needed in a previous version where users could add custom sources through the json field, but not anymore since I removed it. Sorry about that.